Assets/Scripts/Utilities/SoundManager.cs

```diff
@@ -2,254 +2,127 @@
 using System.Collections;
 using System.Collections.Generic;
 
-public class SounderTag : MonoBehaviour {}
-public class SongTag : MonoBehaviour {}
-
-public enum AudioType
+public class SoundManager : SingletonBehaviour<SoundManager>
 {
-	Sound = 0,
-	Song = 1
-}
+	[HideInInspector] public GameObject audioObjHolder = null;
+	[HideInInspector] public GameObject loopObjHolder = null;
+	public List<AudioSource> audioObjs = new List<AudioSource>();
 
-[System.Serializable]
-public class AudioPool
-{
-	public string poolName;
-	public AudioType poolType;
-	public AudioClip[] audioClips;
-	
-	[HideInInspector] public GameObject audioObjHolder;
-	[HideInInspector] public List<AudioSource> audioObjs;
+	AudioSource defaultSource;
 
-	public Dictionary<string, AudioClip> clipList = new Dictionary<string, AudioClip>();
-	public bool muted = false;
+    void Awake()
+    {
+		defaultSource = gameObject.AddComponent<AudioSource>();
+		defaultSource.playOnAwake = false;
 
-	public void SetMuted(bool mute)
-	{
-		muted = mute;
-	}
+		SetupPool( gameObject );
+    }
 
-	public void SetupPool(GameObject soundManagerObj)
+	public void SetupPool( GameObject soundManagerObj )
 	{
 		// Create object to hold sounders
-		audioObjHolder = new GameObject(poolName);
+		audioObjHolder = new GameObject( "AudioPool" );
 		audioObjHolder.transform.parent = soundManagerObj.transform;
+		
+		loopObjHolder = new GameObject( string.Concat( "AudioPool", "_Loops" ) );
```

Comment:
Why are you doing `string.Concat()` and two string literals? Also, using the `+` operator is preferable to `string.Concat()` because it is more readable and they compile to the same code: http://www.dotnetperls.com/string-concat